tests/PactNet.Tests/PactNet.Tests.csproj

```diff
@@ -37,4 +37,9 @@
   <ItemGroup>
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net50'">
```

Comment:
This import should be unnecessary because the package will be pulled in transitively from the referenced main projects.